rules/common/src/main/kotlin/io/nlopez/compose/rules/ComposeModifierNaming.kt

```diff
@@ -19,8 +19,13 @@ class ComposeModifierNaming : ComposeKtVisitor {
 
         val count = modifiers.size
 
-        for (modifier in modifiers) {
-            val valid = modifier.name?.lowercase()?.endsWith("modifier") ?: false
+        modifiers.forEachIndexed { index, modifier ->
+            val isCorrectSingleModifier = modifier.name?.equals("modifier") ?: false
+            val valid = if (index == 0) {
+                isCorrectSingleModifier
+            } else {
+                !isCorrectSingleModifier && modifier.name?.lowercase()?.endsWith("modifier") ?: false
+            }
```

Comment:
You can probably write this in a more straightforward way.

```
if (count == 1) {
  if (modifiers.first().name?.equals("modifier")) {
    emitter.report(modifier, ModifiersAreSupposedToBeCalledModifierWhenAlone)
    return
  }
} else {
  // The old code here
}
```

Reply:
Updated in 1833017